deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -29,7 +31,7 @@ spec:
         - name: efs-plugin
           securityContext:
             privileged: true
-          image: amazon/aws-efs-csi-driver:latest
+          image: "amazon/aws-efs-csi-driver:v1.1.1"
```

Comment:
Do we bump this 1.2?

Reply:
Yeah what I'm planning is

1. create release-1.2 branch
2. push changelog, helm chart changes, etc. to release-1.2 branch
3. tag the HEAD of release-1.2 branch with v1.2.0 / v1.2.0-rc.1
4. after verifying image has pushed etc., cherry-pick the commit from step 2 into master branch.